### ISSUE TITLE
[Bug] buttons in the manage back up section under settings have no labels.

### ISSUE THREAD

**Issue by marrie** [crowbartools/Firebot#2212]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When in the settings tab and [nvda](http://nvda-project.org) is being used, all of the buttons under manage backups are unlabeled.

### Expected Behavior

I should  be able to  manage my backups in an accessible and inclusive manner.

### Steps To Reproduce

* launch NVDA.
* launch firebot
* navigate to settings and press enter.
* use B or control f and search for backup. Once found in the settings section hit enter.
*  find and hit enter on "manage back ups.
* notice that each of the buttons has no label when tabbing or using arrow keys.
This needs to be looked into so this way I can effectively manage my backups and do whatever it is you can do in that section. Here is a screenshot of the affected area. 
![screenshot of the manage backup setting in firebot](https://github.com/crowbartools/Firebot/assets/3614487/452229bb-9762-41f7-a36d-24fc0e16f498)

### Version

Firebot v5.57.0

### What operating system are you using Firebot on?

Windows 11

### Contact Details (Optional)

discord: ke7zum
mastodon: ke7zum@mastodon.cc
